syntax = "proto3";

package video;

service VideoService {
  rpc DeleteVideo(DeleteVideoRequest) returns (DeleteVideoResponse);
  rpc ImportLocalVideos(ImportLocalVideosRequest) returns (ImportLocalVideosResponse);
}

message DeleteVideoRequest {
  string auth_token = 1;
  string video_id = 2;
}

message DeleteVideoResponse {
  bool success = 1;
  string message = 2;
}

message ImportLocalVideosRequest {
  string auth_token = 1;
  string source_dir = 2;
}

message ImportLocalVideosResponse {
  bool success = 1;
  string message = 2;
}